docs/ugcstor.md

```diff
@@ -544,6 +544,139 @@ Note: the deletion of backup will not delete the snapshots created as part of ba
+```
+~$ velero restore get
+NAME                BACKUP    STATUS      WARNINGS   ERRORS    CREATED                         SELECTOR
+s2-20190221142323   s2        Completed   2          1         2019-02-21 14:23:24 +0530 IST   <none>
+```
+
+To delve into the warnings and errors in more detail, you can use velero restore describe:
+```
+velero restore describe s2-20190221142323
+```
+
+The output looks like this:
+```
+Name:         s2-20190221142323
+Namespace:    velero
+Labels:       <none>
+Annotations:  <none>
+
+Backup:  s2
+
+Namespaces:
+  Included:  *
+  Excluded:  <none>
+
+Resources:
+  Included:        *
+  Excluded:        nodes, events, events.events.k8s.io, backups.velero.io, restores.velero.io
+  Cluster-scoped:  auto
+
+Namespace mappings:  <none>
+
+Label selector:  <none>
+
+Restore PVs:  true
+
+Phase:  Completed
+
+Validation errors:  <none>
+
+Warnings:
+  Velero:        <none>
+  Cluster:    <none>
+  Namespaces:
+    litmus:  not restored: persistentvolumeclaims "demo-cstor-vol-claim-ps" already exists and is different from backed up version.
+             not restored: cstorbackupdatas.openebs.io "test2" already exists and is different from backed up version.
+
+Errors:
+  Velero:        <none>
+  Cluster:  <none>
+  Namespaces: <none>
+```
+
+Errors appear for incomplete or partial restores.
+Warnings appear for non-blocking issues (e.g. the restore looks "normal" and all resources referenced in the backup exist in some form, although some of them may have been pre-existing).
+
+Both errors and warnings are structured in the same way:
+
+  Velero: A list of system-related issues encountered by the Velero server (e.g. couldn't read directory).
+
+  Cluster: A list of issues related to the restore of cluster-scoped resources.
+
+  Namespaces: A map of namespaces to the list of issues related to the restore of their respective resources.
+
+Some general commands for troubleshooting that may be helpful:
+`velero backup describe <backupName>` - describe the details of a backup
+`velero backup describe <backupName>` --details - describe the details of a backup and snapshot of persistent volumes
+`velero backup logs <backupName>` - fetch the logs for this specific backup. Useful for viewing failures and warnings, including resources that could not be backed up.
+`velero restore describe <restoreName>` - describe the details of a restore
+`velero restore logs <restoreName>` - fetch the logs for this specific restore. Useful for viewing failures and warnings, including resources that could not be restored.
+`kubectl logs deployment/velero -n velero` - fetch the logs of the velero server pod. This provides the output of the Velero server processes.
+
+You can increase the verbosity of velero server logging by editing Velero deployment file to look like this:
+
+```
+kubectl edit deployment/velero -n velero
```

Comment:
We can separate commands and output separately.. You can mention like "Snippet of output looks like this way"